src/translations.js

```diff
@@ -151,24 +158,41 @@ module.exports = class Translation {
                     }
                 }, function (err) {
                     if (err) {
-                      return err;
+                      return reject(err);
+                    }
+                    else{
+                      return resolve(body);
                     }
                 })
             })
             }
         }
-    });
+        else{
+          return reject("Downloading Translations failed");
+        }
+      })
+    })
   }
 
   spawnTranslations () {
     let self = this;
     let alpha = moment();
+    let counter=0;
       console.info(`Spawning Translations started for ${self.config._LOCALES}`);
       Promise.map(self.config._LOCALES, (translationLocale) => {
-        if(translationLocale.toLowerCase()=="ch"){
-          self.getTranslationsFromLokalise();
+        let cembraLocales=["de","it","fr","en"];
```

Comment:
same here

Reply:
yes, the downloaded file has the name "de.json" so i am saving it in cache based on file name as  "de" .